Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/common/SparkSubmissionParameter.java

```diff
@@ -160,7 +160,7 @@ public void setLocalArtifactPath(String path) {
         localArtifactPath = path;
     }
 
-    @JsonProperty(NAME)
+    @JsonProperty("name")
```

Comment:
Then property NAME can be removed now? After this change, it will be not used by any code.